bids/layout/models.py

```diff
@@ -700,7 +700,10 @@ def _init_on_load(self):
             self.dtype = 'json'
         else:
             self.dtype = type_map[self._dtype]
-            self.value = self.dtype(self._value)
+            if self.dtype == bool and self._value == "False":
+                self.value = False
+            else:
+                self.value = self.dtype(self._value)
```

Comment:
I don't trust that this is the only place where this problem comes up, nor that this is the best way to handle it.